compiler-plugin/src/main/scala/scalapb/compiler/DescriptorImplicits.scala

```diff
@@ -815,6 +815,24 @@ class DescriptorImplicits private[compiler] (
         .map(Helper.escapeComment)
         .filter(_.nonEmpty)
     }
+
+    private[this] def deprecatedAnnotation: Seq[String] = {
+      if (enumDescriptor.getOptions.getDeprecated)
+        List(ProtobufGenerator.deprecatedAnnotation)
+      else
+        Nil
+    }
+
+    def annotationForTraitList: Seq[String] =
+      deprecatedAnnotation ++ scalaOptions.getAnnotationsForTraitList().asScala.toSeq
+
+    def annotationForValueList: Seq[String] =
```

Comment:
Shouldn't this method take the EnumValueDescriptor too so it concats the annotations from the EnumValueDescriptor's options?

Reply:
I was debating pros and cons and ended up leaving like this because the reader can see it better whats happening in the generator:
```
          .add("@SerialVersionUID(0L)")
          .seq(e.valueAnnotationList)
          .seq(v.annotationList)
```

Another thing I was thinking: Is there a scenario when the order of the annotations really matter? Shouldn't right?